cmd/podman/spec.go

```diff
@@ -83,18 +71,6 @@ func addNetNS(config *createConfig, g *generate.Generator) error {
 		return nil
 	} else if netMode.IsContainer() {
 		logrus.Debug("Using container netmode")
```

Comment:
????  Does nothing?

Reply:
The logic is handled when we generate the libpod options. I could move the container lookups back into here and store the containers whose namespaces we want to join in createConfig, but keeping the logic all in one place seemed simpler

Reply:
Yes I realized after reading the patch.  Could you add a comment here explaining where the IsContainer is handled.  So future engineers will understand.